app/views/tutors/_form.html.erb

```diff
@@ -1,118 +1,186 @@
 
-  <% if @tutor.errors.any? %>
-    <div id="error_explanation">
-      <h2>
-        <%= pluralize(@tutor.errors.count, "error") %> prohibited
-        this tutor from being saved:
-      </h2>
-      <ul>
-        <% @tutor.errors.full_messages.each do |msg| %>
-          <li><%= msg %></li>
-        <% end %>
-      </ul>
-    </div>
-  <% end %>
-
-  <div class="row">
-    <div class="col-md-3">
-      <%= f.input :first_name %>
-    </div>
-    <div class="col-md-4">
-      <%= f.input :last_name %>
-    </div>
-    <div class="col-md-3">
-      <%= f.input :dob, as: :date, label: "Date of Birth",
-                                   start_year: Date.today.year - 70,
-                                   end_year: Date.today.year - 12 %>
+<% if @tutor.errors.any? %>
+  <div id="error_explanation">
+    <h2>
+      <%= pluralize(@tutor.errors.count, "error") %> prohibited
+      this tutor from being saved:
+    </h2>
+    <ul>
+      <% @tutor.errors.full_messages.each do |msg| %>
+        <li><%= msg %></li>
+      <% end %>
+    </ul>
+  </div>
+<% end %>
+
+<form role="form">
+
+  <div class="row">
+    <div class="col-md-6">
+      <!-- Affiliate will go here --!>
```

Comment:
Careful here, you never closed the comment so the rest of the file is commented out!

`<!-- my comment -->` (no second bang)